### ISSUE TITLE
Latest release broke webpack

### ISSUE THREAD

**Issue by mcmonster** [xtuc/webassemblyjs#450]
```
npm ERR! code ETARGET
npm ERR! notarget No matching version found for @webassemblyjs/ast@1.7.6
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
npm ERR! notarget 
npm ERR! notarget It was specified as a dependency of 'webpack'
npm ERR! notarget 
```

Webpack version is `"webpack": "^4.16.5",`

**Comment by mcmonster** [comment]
Okay, noticing now that it appears you are deleting the `1.7.6` version of the `webassemblyjs` submodules and reissuing them - presumably with some bug fix you are backporting.

This is breaking my CI pipelines, and presumably is doing so for many others with the `webpack` dependency. In the future, might I suggest instead incrementing the version?

**Comment by mrosshcm** [comment]
My error is for `@webassembly/helper-wasm-bytecode@1.7.6` but otherwise the same.

**Comment by mcmonster** [comment]
@mrosshcm I suspect it will occur with any of the various `@webassembly/` sub-packages. It appears their release procedure is replacing all of them.

**Comment by xtuc** [comment]
https://www.npmjs.com/package/@webassemblyjs/ast/v/1.7.6 or https://www.npmjs.com/package/@webassemblyjs/helper-wasm-bytecode/v/1.7.6 exists on npm

**Comment by xtuc** [comment]
On my side installing webpack@4.18.0 is fine. Are you using a npm proxy/cache?

**Comment by mrosshcm** [comment]
@xtuc Of course, it was a caching problem. Running `npm cache verify` then installing again worked for me.

@mcmonster ?

**Comment by mcmonster** [comment]
As someone who consumes these packages far up the food chain, I'm curious why it is that a version increment isn't used for these updates? It seems a little unreasonable that your package updates would break consumers sitting multiple levels up in the dependency tree, even if it's only for a small period of time.

Webpack is used by a vast number of devs - surely even small periods of breakage will affect a decent chunk of people, no?

**Comment by sokra** [comment]
I tried to figure out how this could happen. Seems like webpack was released ~15minutes after `@webassemblyjs/ast` was released. Looks like this was not enough time and existing cache entries for these packages are still considered valid. Unfortunately I didn't find any info how long this timespan is, resp. how long one should wait after a dependency release until the parent release...

The npm registry documentation states a max-age of 5 min, but this doesn't seem to be true. Probably the npm cli also accept cache entries for a while without revalidation, or the cloudflare server seem to have some internal cache.
